metricbeat/module/prometheus/collector/_meta/testdata/etcd-3.3.10-partial.plain-expected.json

```diff
@@ -11,11 +11,11 @@
         },
         "prometheus": {
             "labels": {
-                "instance": "127.0.0.1:51051",
-                "server_id": "8e9e05c52164694d"
+                "instance": "127.0.0.1:58013",
+                "server_version": "3.3.10"
```

Comment:
These changes also happened when running `go test -generate` on master branch without this change.